### PR TITLE
Use a variable for the target column as in the other examples, fix typo

### PR DESCRIPTION
**Pull request by kinow**
Hi, found one typo while reading the code, and one example in the intro chapter that didn't use the target column.

I tried running the Python script, but that didn't modify the Notebook file? I couldn't find instructions on how to re-generate the notebooks, so if anyone could direct me to how to generate these, I can then amend this PR :+1: 

Thanks!!! :nerd_face:

### CODE REVIEW

File: python_scripts/01_tabular_data_exploration.py [hunk 1]
```diff
@@ -245,7 +245,7 @@
     x="age",
     y="hours-per-week",
     data=adult_census[:n_samples_to_plot],
-    hue="class",
+    hue=target_column,
```
Comment (ArturoAmorQ) [inline python_scripts/01_tabular_data_exploration.py:248]: Could you please make the same modification in the scatterplot just below this one (line 269)?

**Review by ArturoAmorQ — APPROVED** [review]
Apart from small comment LGTM

**Comment by ArturoAmorQ** [comment]
> I couldn't find instructions on how to re-generate the notebooks, so if anyone could direct me to how to generate these, I can then amend this PR

Don't worry, we will build the notebooks ourselves afterwards.

File: python_scripts/01_tabular_data_exploration.py (hunk 1 above)
Comment (kinow) [reply python_scripts/01_tabular_data_exploration.py:248]: Done, thanks @ArturoAmorQ !

**Review by ArturoAmorQ — APPROVED** [review]
LGTM, Thanks for the contribution @kinow :)

**Pull request merged**